jrpc/client.go

```diff
@@ -99,13 +99,24 @@ func (m *jrpcManager) Close(ctx context.Context) error {
 	if err != nil {
 		return errors.WithStack(err)
 	}
-	if resp.Succuess {
+	if resp.Success {
 		return nil
 	}
 
 	return errors.New(resp.Text)
 }
 
+func (m *jrpcManager) DownloadFile(ctx context.Context, url string, path string) error {
+	resp, err := m.client.DownloadFile(ctx, &internal.DownloadInfo{Url: url, Path: path})
+	if err != nil {
+		return errors.WithStack(err)
+	}
+	if resp.Success {
+		return nil
+	}
+	return errors.New(resp.Text)
+}
```

Comment:
I think we don't need this here. the jrpcManager type is exported only as an implementation of jasper.Manager, which means this isn't exported.

This operation will be available to clients who are using the generated grpc client (stub), so it's fine to just remove this, I think.

Reply:
Removed it.